code/system/application/legacy/vanishingpoint/include/VanishingPointDetection.h

```diff
@@ -0,0 +1,268 @@
+#ifndef LANE_MARKINGS_DETECTION_H_
```

Comment:
Once you have adjusted the sources, please run https://github.com/chalmers-revere/opendlv/blob/master/scripts/refactorCWD.sh on your sub-tree to let clang reformat your sources to follow our coding style.